_data/chains/eip155-3601.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Pandonet",
+  "chain": "Pando",
+  "rpc": ["https://eth-rpc-api.pandoproject.org/rpc"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Pando token ",
+    "symbol": "PTX",
+    "decimals": 18
+  },
+  "infoURL": "https://www.pandoproject.org/",
+  "shortName": "pando-mainnet",
+  "chainId": 3601,
+  "networkId": 3601,
+  "explorers": [
+    {
+      "name": "Pando Mainnet Explorer",
+      "url": "https://explorer.pandoproject.org",
```

Comment:
does not seem to be EIP3091 compatible - e.g. see:

https://explorer.pandoproject.org/block/4457685

Reply:
Hello @ligi https://explorer.pandoproject.org/blocks/4457685 is the correct url

Reply:
please refer this url = https://explorer.pandoproject.org/blocks/4457685     , the one you sent is not correct

Reply:
that violates EIP3091

Reply:
that is fixed now
https://explorer.pandoproject.org/block/4459713
please review

Reply:
EIP3091 rules are followed please review and update @ligi